### DISCUSSION TITLE
Jump to a specific time while in review page

### DISCUSSION THREAD

**Discussion by 18923** [blakeblackshear/frigate#14957] [Ask A Question]
Instead of scrolling can you enter a specific time in the review page? If not can this be added in the future.

![Blank](https://github.com/user-attachments/assets/16ab38ef-7da1-4b5e-a747-80a7d2abf98c)

**Comment by NickM-27** [comment]
you can click anywhere on the timeline to jump to that time, you can scroll using a mousewheel

**Reply by 18923** [reply]
**re: (_you can click anywhere on the timeline to jump to that time_)**

Yes you can but most of the timeline is not visible and it's very slow to scroll down a few hours. My laptop has no scroll wheel, I have to pin the pointer to the bottom of the timeline and wait for the desired time to slowly scroll up. If time of interest could be entered directly it would help.

**Reply by NickM-27** [reply]
you can just use the two fingers to scroll on the trackpad

**Reply by Imwithstupid007** [reply]
You could also use CTRL-scroll on the time line. It will compress the time line and you can scroll trough larger time blocks faster.
